databricks-sdk-java/src/test/java/com/databricks/sdk/integration/ClustersIT.java

```diff
@@ -47,4 +48,14 @@ void latestRuntime(WorkspaceClient w) {
 
     assertNotNull(runtime);
   }
+
+  @Test
+  void clusterDoesNotExist(WorkspaceClient w) {
+    assertThrowsExactly(
```

Comment:
This test will fail once I merge the follow-up change (equivalent of  https://github.com/databricks/databricks-sdk-py/commit/45a356ca24ce92c80f06c3f60f9969aaf25987e4)

Reply:
ack, thanks

Reply:
I will fix it in that PR, no worries.l